openregister-conformance

```diff
@@ -10,13 +10,13 @@ import pytest
 
 parser = argparse.ArgumentParser()
 parser.add_argument("endpoint", nargs="+", help="URL(s) of register to test")
+parser.add_argument("--register_register_endpoint", default="https://register.register.gov.uk", help="URL of register register")
```

Comment:
I have no specific choice as such, underscore is python's naming convention for a variable.